serverless.test.yml

```diff
@@ -1,6 +1,380 @@
+    body:
+      id: 144444
+      fname: updateTest
+      lname: dudeGotUpdated
+      faculty: Sauder
+
+- name: user update no ID returns 406
+  endpoint:
+    method: POST
+    path: users/update
+  request:
+    body:
+      fname: badUpdateTest
+      lname: dudeNotUpdated
+      faculty: LFS
+  response:
+    status: 406
+    body:
+      message: "User ID not specified."
+
+- name: user update not found returns 404
+  endpoint:
+    method: POST
+    path: users/update
+  request:
+    body:
+      id: -20
+      fname: badUpdateTest
+      lname: dudeNotUpdated
+  response:
+    status: 404
+    body: "\"User not found.\""
+
+- name: event create returns 200
+  endpoint:
+    method: POST
+    path: events/create
+  request:
+    body:
+      id: testevent
+      ename: test
+      capac: 2000000
+      img: yup
+      code: ripp
+  response:
+    status: 200
+    body:
+      message: 'Event Created!'
+
+- name: event create no ID returns 406
+  endpoint:
+    method: POST
+    path: events/create
+  request:
+    body:
+      ename: test
+      capac: 2000000
+      img: yup
+  response:
+    status: 406
+    body:
+      message: 'Event ID not specified.'
+      data:
+        ename: test
+        capac: 2000000
+        img: yup
+
+- name: event create no capac returns 406
+  endpoint:
+    method: POST
+    path: events/create
+  request:
+    body:
+      id: testevent
+      ename: test
+      img: yup
+      code: test
+  response:
+    status: 406
+    body:
+      message: 'Capacity invalid, please provide valid number.'
+      data:
+        id: testevent
+        ename: test
+        img: yup
+
+- name: event get returns 200
+  endpoint:
+    method: GET
+    path: events/get
+  response:
+    status: 200
+
+- name: event update returns 200
+  endpoint:
+    method: POST
+    path: events/update
+  request:
+    body:
+      id: testevent
+      ename: testUpdated
+      capac: 7
+      img: nope
+      code: test
+  response:
+    status: 200
+    body: "\"Update succeeded.\""
+
+- name: event update not found returns 404
+  endpoint:
+    method: POST
+    path: events/update
+  request:
+    body:
+      id: nonexistanttestidwoo
+      ename: testUpdated
+      capac: 7
+      img: nope
+      code: test
+  response:
+    status: 404
+    body: "\"Event not found.\""
+
+- name: event update no ID returns 406
+  endpoint:
+    method: POST
+    path: events/update
+  request:
+    body:
+      ename: testUpdated
+  response:
+    status: 406
+    body:
+      message: 'Event ID not specified.'
+
+- name: event scan match returns 200
+  endpoint:
+    method: GET
+    path: events/scan
+  request:
+    form:
+      code: test
+  response:
+    status: 200
+    body:
+      size: 1
+
+- name: event scan no match returns 200
+  endpoint:
+    method: GET
+    path: events/scan
+  request:
+    form:
+      code: -1
+  response:
+    status: 200
+    body:
+      size: 0
+
+- name: registration create returns 200
```

Comment:
this file is getting long, do you think we should split into multiple files? (not sure if thats possible)

Reply:
Not sure either, will look into it

Reply:
Can't seem to find a workaround within serverless, and YAML doesn't directly support importing from other YAML files, could use something like this tho: https://www.npmjs.com/package/yaml-import

Reply:
For now I guess a longer file is simpler than using that package. We can reevaluate if it starts causing problems